repos/core-setup.proj

```diff
@@ -6,13 +6,12 @@
     <BuildArguments Condition="$(Platform.Contains('arm'))">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
     <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) -- /p:BuildDebPackage=false</BuildCommand>
     <BuildCommand Condition="$(Platform.Contains('arm'))">$(ArmEnvironmentVariables) $(BuildCommand)</BuildCommand>
-    <PackagesOutput>$(ProjectDirectory)/Bin/$(TargetRid).$(Configuration)/packages/</PackagesOutput>
-    <BinariesOutput>$(PackagesOutput)</BinariesOutput>
     <LatestCommit>82b22fa44e46d624ebeddf156f33e9cfb317bd57</LatestCommit>
     <VersionSeedDate>2017-07-20</VersionSeedDate>
     <OfficialBuildId>20170720-05</OfficialBuildId>
-    <DependencyVersionInputRepoApiImplemented>false</DependencyVersionInputRepoApiImplemented>
-    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
+
+	<!-- Need to set $(PackagesOutput) so WriteVersions writes the versions file for cli, until cli respects auto-dependency flow -->
+    <PackagesOutput>$(SourceBuiltPackagesPath)</PackagesOutput>
```

Comment:
Tab char snuck in before the comment.

Reply:
I fixed it, since I was updating doc anyway.